src/test/java/com/jcabi/dynamo/mock/H2DataTest.java

```diff
@@ -134,7 +134,26 @@ public void createsTablesWithLongNames() throws Exception {
      */
     @Test
     public void supportsTableNamesWithIllegalCharacters() throws Exception {
-        new H2Data().with(".-.", new String[] {"pk"}, new String[0]);
+        new H2Data().with(".-.", new String[]{"pk"}, new String[0]);
+    }
+
+    /**
+     * H2Data supports column names with characters illegal to H2.
+     * todo: #28 user reports that H2Data does not support
+     * COLUMNS with ".", "-", or digits but i don't know for sure
```

Comment:
issue creator expects <code>H2Data</code> supports such symbols. But i'm not really sure that it's correct. How to deal with that in this case?